src/Modules/Backend/assets/Backend/webpack/js/Components/Modal.js

```diff
@@ -7,11 +7,11 @@ export class Modal {
     }
 
     $modals.on('shown.bs.modal', (e) => {
-      $('#ajaxSpinner').addClass('light')
+      $('#ajax-spinner').addClass('light')
```

Comment:
@carakas I have a task for @daphneslootmans to cleanup CSS and JS when fork6 is a bit further. Let's focus here on the CI issue, otherwise this PR will get out of focus.